packages/hoppscotch-common/src/helpers/editor/extensions/HoppEnvironment.ts

```diff
@@ -123,11 +128,16 @@ const cursorTooltipField = (aggregateEnvs: AggregateEnvironment[]) =>
             invokeActionType = "modals.my.environment.edit"
           }
 
-          invokeAction(invokeActionType, {
-            envName: tooltipEnv?.sourceEnv !== "Global" ? envName : "Global",
-            variableName: parsedEnvKey,
-            isSecret: tooltipEnv?.secret,
-          })
+          if (tooltipEnv?.sourceEnv === "RequestVariable") {
+            restTabs.currentActiveTab.value.document.optionTabPreference =
+              "requestVariables"
+          } else {
+            invokeAction(invokeActionType, {
+              envName: tooltipEnv?.sourceEnv !== "Global" ? envName : "Global",
```

Comment:
```suggestion
              envName: tooltipEnv?.sourceEnv === "Global" ? "Global" : envName ,
```

Let's check for the positive condition.